.circleci/config.yml

```diff
@@ -389,6 +204,11 @@ workflows:
       - vrt_test:
           requires:
             - build
+          filters:
+            branches:
+              ignore:
+                - master
```

Comment:
As mentioned here (https://github.com/commercetools/merchant-center-application-kit/issues/767#issuecomment-503512937) we should always run Percy on PRs.